topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -662,39 +697,263 @@ In the previous section, we counted reads mapped to genes for two samples. To be
+
+Let's imagine we have RNA-seq counts from 3 samples for a genome with 4 genes:
+
+Gene | Sample 1 Counts | Sample 2 Counts | Sample 3 Counts
+--- | --- | --- | ---
+A (2kb) | 10 | 12 | 30
+B (4kb) | 20 | 25 | 60
+C (1kb) | 5 | 8 | 15
+D (10kb) | 0 | 0 | 1
+
+Sample 3 has more reads than the other replicates, regardless of the gene. It has a higher sequencing depth than the other replicates. Gene B is twice as long as gene A: it might explain why it has twice as many reads, regardless of replicates
+
+The number of sequenced reads mapped to a gene depends then on:
+
+- The **sequencing depth** of the samples
+
+    Samples sequenced with more depth will have more reads mapping to each genes
+
+- The **length of the gene**
+
+    Longer genes will have more reads mapping to them
+
+To compare samples or gene expressions, the gene counts need to be normalized. We could use TPM (Transcripts Per Kilobase Million). 
+
+> ### {% icon details %} RPKM, FPKM and TPM?
+>
+> These three metrics are used to normalize count tables for:
+> - sequencing depth (the "Million" part)
+> - gene length (the "Kilobase" part)
+>
+> Let's use the previous example to explain RPK, FPKM and TPM.
+>
+> For **RPKM** (Reads Per Kilobase Million),
+> 1. Compute the "per million" scaling factor: sum up the total reads in a sample and divide that number by 1,000,000
+>
+>    Gene | Sample 1 Counts | Sample 2 Counts | Sample 3 Counts
+>    --- | --- | --- | ---
+>    A (2kb) | 10 | 12 | 30
+>    B (4kb) | 20 | 25 | 60
+>    C (1kb) | 5 | 8 | 15
+>    D (10kb) | 0 | 0 | 1
+>    **Total reads** | 35 | 45 | 106
+>    **Scaling factor** | 3.5 | 4.5 | 10.6
+>
+>    *Because of the small values in the example, we are scoring using a factor of 10.*
+>    
+> 2. Divide the read counts by the "per million" scaling factor
+>
+>    This normalizes for sequencing depth, giving reads per million (RPM)
+>
+>    Gene | Sample 1 RPM | Sample 2 RPM | Sample 3 RPM
+>    --- | --- | --- | ---
+>    A (2kb) | 2.86 | 2.67 | 2.83
+>    B (4kb) | 5.71 | 5.56 | 5.66
+>    C (1kb) | 1.43 | 1.78 | 1.43
+>    D (10kb) | 0 | 0 | 0.09  
+> 
+> 3. Divide the RPM values by the length of the gene, in kilobases
+>
+>    Gene | Sample 1 RPKM | Sample 2 RPKM | Sample 3 RPKM
+>    --- | --- | --- | ---
+>    A (2kb) | 1.43 | 1.33 | 1.42
+>    B (4kb) | 1.43 | 1.39 | 1.42
+>    C (1kb) | 1.43 | 1.78 | 1.42
+>    D (10kb) | 0 | 0 | 0.009  
+>
+> **FPKM** (Fragments Per Kilobase Million) is very similar to RPKM. RPKM is for single-end RNA-seq and FPKM for paired-end RNA-seq. With single-end, every read corresponds to a single fragment that was sequenced. With paired-end RNA-seq, a single fragment are the two reads of a pair, or if one read in the pair did not map, one read can correspond to a single fragment (in case we decided to keep them). FPKM keeps tracks of fragments so that one fragment with 2 reads is counted only once.
```

Comment:
```suggestion
> **FPKM** (Fragments Per Kilobase Million) is very similar to RPKM. RPKM is for single-end RNA-seq and FPKM for paired-end RNA-seq. With single-end, every read corresponds to a single fragment that was sequenced. With paired-end RNA-seq, two reads of a pair are mapped from a single fragment, or if one read in the pair did not map, one read can correspond to a single fragment (in case we decided to keep these). FPKM keeps tracks of fragments so that one fragment with 2 reads is counted only once.
```